validator/bigquery_cmek.rego

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPBigqueryCMEKEncryptionConstraintV1
+
+import data.validator.gcp.lib as lib
+
+###########################
+# Find Whitelist Violations
```

Comment:
Not sure what this comment is trying to say? Can we either expand it or delete it?

Reply:
Deleted the comment as the yaml file already have the description of the constraint.